paddlecloud/notebook/tls.py

```diff
@@ -36,6 +36,9 @@ def create_user_cert(ca_path, username):
         %s/%s.csr -subj \"/CN=%s\""%\
         (user_cert_dir, username,
         user_cert_dir, username, username))
+    # FIXME(gongwb):why need delete ca.srl when mount afs path while not need
```

Comment:
FIXME(gongwb): The file `ca.srl` containing a serial number will cause an error on the AFS (mount local path by VFS ), delete this file for temporary and make the code work!